ios-target-app/Extensions/TextFieldExtension.swift

```diff
@@ -15,14 +15,21 @@ extension UITextField {
         backgroundColor: UIColor = .white,
         height: CGFloat = UI.TextField.height,
         borderStyle: BorderStyle = .line,
+        pickerView: UIPickerView? = nil,
```

Comment:
I think this is a very rare scenario and can be done separately to keep this function as small as possible and with the basic/common configurations only.
This is a code smell as well, probably Swiftlint needs to be configured properly for the `function_parameter_count` rule to detect this cases.